src/efu/Parser.cpp

```diff
@@ -70,6 +70,45 @@ static int stat_get(std::vector<std::string> cmdargs, char *output,
   return Parser::OK;
 }
 
+//=============================================================================
```

Comment:
Would be nice to have a  \brief description about these functions. If you write one, I can generate similar ones for the others with copilot.

Reply:
ok